packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -1132,14 +1132,14 @@ function mountDeferredValue<T>(
   value: T,
   config: TimeoutConfig | void | null,
 ): T {
-  const [prevValue, setValue] = mountState(value);
+  const [prevValue, setValue] = mountState(() => value);
```

Comment:
If you use `mountReducer` and `updateReducer` instead, then you can write your own version of `basicStateReducer` and avoid these extra closures

Reply:
(where by "your own version of `basicStaticReducer`" I mean "a hoisted identity function" :D)